Doc set for VMFR (#110937)
openshift/openshift-docs: modules/oadp-vmfr-custom-resources.adoc

// Module included in the following assemblies:
//
// backup_and_restore/application_backup_and_restore/oadp-vmfr/oadp-vmfr.adoc

:_mod-docs-content-type: REFERENCE
[id="oadp-vmfr-custom-resources_{context}"]
= {oadp-short} VMFR custom resources

[role="_abstract"]

> Use {oadp-short} VMFR custom resources to discover VM backups and restore individual files from those backups. This provides a file-level recovery mechanism for `kubevirt` virtual machines.

Reviewer: ```suggestion
Use {oadp-short} virtual machine file restore (VMFR) custom resources to discover VM backups and restore individual files from those backups. You can perform file-level recovery for `kubevirt` VMs.
```
